proposal.rst

```diff
@@ -105,48 +104,46 @@ Background Information
 similar programmes before?  If so, tell us about how it went, and any areas you
 would have liked more help with.**
 
-FILLME
+	I have not take part in such projects.
 
 **Please tell us about any previous experience you have with Xapian, or other
 systems for indexed text search.**
 
-FILLME
+	I didn't have enough time to contribute because I just met with Xapian. I will make good use of the time up to the student selection process and contribute to Xapian.
 
 **Tell us about any previous experience with Free Software and Open Source
 other than Xapian.**
 
-FILLME
+	I joined Hacktoberfest.
```

Comment:
And?

What did you do?  What did you learn?  What was good and what was bad about it?